CODE_OF_CONDUCT.md

```diff
@@ -73,8 +85,18 @@ affect a person's ability to participate within them.
   bad day, so please take this opportunity to question yourself, cool down if
   necessary and do not persist in the exact same behavior you have been
   reported for.
-
-
+- **Avoid using personal (member-to-member) communication.** 
+  Personal communication channels (e.g., direct messages on Zulip or
+  personal emails) limit the visibility of interactions for other users and
+  contributors, preventing them from benefiting from shared
+  knowledge. Additionally, unsollicted private communication can
+  sometimes be perceived as too intrusive.
```

Comment:
I'd like to see some process in place for community members who are not comfortable being public with initial concerns, e.g., either restricting this to "repeated unsolicited private communication", or saying something like "if you have a concern you're uncertain about sharing publicly, you can reach out to people XYZ"

Reply:
What kind of concerns do you have in mind? Because obviously, to report problematic behaviors, you should not do it in public, but using the coq-conduct mailing list instead. So, probably, you mean something else?

Reply:
Ah, sorry, I had just missed reading the part of the surrounding text that mentioned the coq-conduct mailing list.